src/main/java/application/bookstore/model/Role.java

```diff
@@ -0,0 +1,40 @@
+package application.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.security.core.GrantedAuthority;
+
+@Getter
+@Setter
+@EqualsAndHashCode(of = "id")
+@Entity
+@Table(name = "roles")
+public class Role implements GrantedAuthority {
+    private static final String ROLE = "ROLE_";
```

Comment:
```suggestion
    private static final String PREFIX = "ROLE_";
```
better name after the function that constant serves in our class, not by its value